src/Illuminate/Auth/Notifications/VerifyEmail.php

```diff
@@ -61,7 +61,7 @@ protected function verificationUrl($notifiable)
         return URL::temporarySignedRoute(
             'verification.verify',
             Carbon::now()->addMinutes(Config::get('auth.verification.expire', 60)),
-            ['id' => $notifiable->getKey()]
+            ['id' => $notifiable->getKey(), 'hash' => sha1($notifiable->email)]
```

Comment:
How do we know the email property exists. It's not on the contract.

Reply:
This still just access a property directly?

Reply:
And that's fixed now as well. My apologies.